docs/best-practices.md

```diff
@@ -0,0 +1,132 @@
+---
+slug: /best-practices
+---
+
+# Best practices
+
+## Enable overflow checks
+
+It's usually helpful to panic on integer overflow. To enable it, add the following into your `Cargo.toml` file:
+
+```toml
+[profile.release]
+overflow-checks = true
+```
+
+## Use `assert!` early
```

Comment:
do you think it's worth noting the change to require in this section, I updated `HELP.md` in https://github.com/near/near-sdk-rs/pull/514/files, maybe a modified version where it's noted that you can use `require!` as of `4.0.0-pre.2`?  Fine if not and we can circle back, but I feel like this will be forgotten

Reply:
oh yeah! good point. I have added that now. https://github.com/near/sdk-docs/pull/37/commits/429289e87e8754741336af8ee4167580ec94412b